### ISSUE TITLE
Travel to another continent than Europe

### ISSUE THREAD

**Issue by F1restarter2000** [iam4x/pokemongo-webspoof#326]
Hey!
I've got a a problem. I can only travel in Europe. If I try to go to another continent, the program shows me: "Error with schedule trip - ZERO_RESULTS".

Can someone help me? ^-^

**Comment by thunderboltsid** [comment]
Agreed. If I want to teleport to another continent, it shows up as "ZERO_RESULTS". See screenshot for more info
![screen shot 2016-08-02 at 12 43 43](https://cloud.githubusercontent.com/assets/6081171/17326046/2739546a-58af-11e6-8dc0-15b3ed4b2149.png)

**Comment by Anilo1990** [comment]
Same here

**Comment by Dayce** [comment]
I think it's because there is no road from Europe to USA. I'm living on an island and I cannot go "outside".

**Comment by pcmore** [comment]
Why don't you guys just look in the issues? There is already mentioned a solution.

**Comment by nickg819** [comment]
@pcmore because people don't bother reading the other posts or using search. It's irritating af.

**Comment by jnrepo** [comment]
google where you want to do, find out its long and latitude, use that to teleport
